cucumber/api/features/step_definitions/response_steps.rb

```diff
@@ -53,5 +53,5 @@
 Then(/^the binary result is preserved$/) do
   expect(@result).to be
   expect(@result.headers[:content_type]).to include("application/octet-stream")
```

Comment:
Can we check to make sure `content_type` is exactly `application/octet-stream`, here and in the step above? It shouldn't ever be anything else, but let's not give RestClient the chance to muck with the headers.